stablehlo/tests/vhlo_to_version_attrtype_invalid.mlir

```diff
@@ -10,97 +10,116 @@
 
 
 // This simulates version validation if a new numeric type is introduced.
```

Comment:
We probably also want to have negative tests for invalid function types?

Reply:
Good catch, turns out we don't validate function types. Fixed and tested